### PR TITLE
Toxins cleanup, Isle 3

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Renames most (hopefully all) in-game mentions of toxins as referring to ordnance. ~~I've tried to avoid changing code paths cause I was concerned about breaking something, so that can come at later date if anyone else wants to do it.~~ I've decided if this is to have any actual point to it it should at least change the code accordingly, else it's a mess.

This change includes, but may not be limited to:

- Toxins special being renamed to Ordnance Special, along with the icon state names for the sprites
- Airlock names for Kilo, Tram and Derilect
- Code comments for the research.dm file
- The name of the "Guide to Toxins" manual
- The description for the Nakamura Engineering B.O.M.B Suit, whatever the fuck that is. 
- Two of the cutesy pickup lines of valentine cards

## Why It's Good For The Game

~~Toxins bad, ordnance good.~~
~~I rest my case.~~

On a more serious note than last time, the point is consistency. It's not great to have two words that refer to the same reality  co-exist in the code like this, specially when one of them is as vague as "toxins", so I tried to clean up some of the mentions of it that were missed or intentionally ignored on the first PR that did this.

## Changelog

:cl:
spellcheck: Removes mentions of "toxins" in favour of "ordnance" for several items.
/:cl:

### CODE REVIEW

File: code/game/objects/items/manuals.dm [hunk 1]
```diff
@@ -411,7 +411,7 @@
 	icon_state = "book6"
 	author = "Cuban Pete"
 	title = "Ordnance for Dummies or: How I Learned to Stop Worrying and Love the Maxcap"
-	page_link = "Guide_to_toxins"
```
Comment (JohnFulpWillard) [inline code/game/objects/items/manuals.dm:414]: This is reflected on the wiki right?

Comment (FernandoJ8) [reply code/game/objects/items/manuals.dm:414]: UUuuuuhhhhh

**Comment by FernandoJ8** [comment]
Also jeez this caused more red checks than I was expecting, better to draft for the moment.

File: code/__DEFINES/research.dm [hunk 2]
```diff
@@ -30,16 +30,15 @@
...
 #define TECHWEB_BOMB_MIN_RANGE (GLOB.MAX_EX_LIGHT_RANGE)
-/// The range required to produce the maximum amount of cash from toxins experiments.
+/// The range required to produce the maximum amount of cash from ordnance experiments.
 #define TECHWEB_BOMB_MAX_RANGE 300
-/// A constant used to scale the cash produced by toxins experiments.
+/// A constant used to scale the cash produced by ordnance experiments.
 #define TECHWEB_BOMB_SCALE_CONST 300000
-/// A constant used to scale the cash produced by toxins experiments.
-#define TECHWEB_BOMB_SCALE_DIVISOR 1500
+/// A constant used to scale the cash produced by ordnance experiments.t#define TECHWEB_BOMB_SCALE_DIVISOR 1500
```
Comment (JohnFulpWillard) [inline code/__DEFINES/research.dm:41]: You accidentally moved the define into the comment

Comment (FernandoJ8) [reply code/__DEFINES/research.dm:41]: Yep yep yep, fixing it

**Comment by FernandoJ8** [comment]
Also I'll hold off on the page move until one of the reviewers gives me the greenlight on moving the Guide to Toxins page to Guide to Ordnance

File: strings/valentines.json [hunk 3]
```diff
@@ -82,7 +82,7 @@
 		"Your face gets me higher than omega weed.",
 		"Someone check the tachyon doppler array, because you just rocked my world.",
 		"I wish I were your family heirloom so I could be with you all the time.",
-		"How much damage does a toxins bomb do? Enough to break the ice.",
+		"How much damage does a ordnance bomb do? Enough to break the ice.",
```
Comment (MrMelbert) [inline strings/valentines.json:85]:
```suggestion
		"How much damage does an ordnance bomb do? Enough to break the ice.",
```
these just sound wrong as "ordnance" anyways

**Comment by SuperNovaa41** [comment]
whats the point

File: strings/valentines.json (hunk 3 above)
Comment (JohnFulpWillard) [reply strings/valentines.json]: maybe TTV instead?

Comment (FernandoJ8) [reply strings/valentines.json]: Hm, that could work.

**Comment by FernandoJ8** [comment]
Updated the description to be more serious and actually informative

**Comment by FernandoJ8** [comment]
Eh, honestly this is too much of a mess and needs groundwork on the wiki and such before it can be done. Will close it for now

**Pull request closed without merging**